dataprofiler/profilers/utils.py

```diff
@@ -343,3 +343,63 @@ def biased_kurt(df_series):
 
     kurt = n * M4 / M2 ** 2 - 3
     return kurt
+
+def find_diff_of_numbers(stat1, stat2):
+    """
+    Finds the difference between two stats. If there is no difference, returns
+    "unchanged". For ints/floats, returns stat1 - stat2. For strings, returns
+    list containing [stat1, stat2]. For lists/sets, removes duplicates and
+    returns [unique values of stat1, shared values, unique values of stat2].
+
+    :param stat1: the first statistical input
+    :type stat1: Union[int, float, str, list, set]
+    :param stat2: the second statistical input
+    :type stat2: Union[int, float, str, list, set]
+    :return: the difference of the stats
+    """
+    diff = "unchanged"
+    if stat1 is None or stat2 is None:
```

Comment:
If both are None this will return `[None, None]` but shouldn't we want "unchanged" in that case?

Reply:
good catch, fixing